src/common/bn055_imu.cc

```diff
@@ -77,14 +77,11 @@ Eigen::Vector3f BN055::getVector(VectorType vectorType)
     // Return vector, scaling correctly based on vector type
     if(vectorType == VectorType::MAGNETOMETER
         || vectorType == VectorType::GYROSCOPE
-        || vectorType == VectorType::EULER)
-    {
-        return Eigen::Vector3f((float)raw[0] / 16.0f, (float)raw[1] / 16.0f, (float)raw[2] / 16.0f);
-    }
-    else {
-        return Eigen::Vector3f((float)raw[0] / 100.0f, (float)raw[1] / 100.0f, (float)raw[2] / 100.0f);
+        || vectorType == VectorType::EULER) {
```

Comment:
This is horrible formatting - I used an extra line break after a multi-line if for a reason

Reply:
It is indeed :laughing: